Tests/travis/README_trailing_spaces.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+functions="$(dirname $0)/functions.sh"
+if [ -f "$functions" ]; then
+  . "$functions"
+else
+  echo "functions file '$functions' not found!"
+  exit 1
+fi
+
+e=0
+while read f; do
+  [ -z "$f" ] && continue
+  r=`getAddedLines "$f" | grep -e '^.*\s\+$'`;
```

Comment:
Won\`t `grep -e '\s$'` work?
I mean, single trailing space must already be enough to say that the line is corrupted.

Reply:
Thank you, regexp was simplified.